broadinstitute/pearl
# [JN-474] Fetch status from Pepper (DSM) by kit ID or study

This may not complete [JN-474](https://broadworkbench.atlassian.net/browse/JN-474) since I haven't yet been able to do any integration testing against Pepper. However, I'm getting anxious to get this code merged. It shouldn't require much if any additional changes, though I'll probably update the integration test before calling JN-474 done; expect one more PR for this ticket.

To test:
At this point, just run the test suite and see that it passes. Once we have more integration and UI built out, we'll be able to do better testing.

[JN-474]: https://broadworkbench.atlassian.net/browse/JN-474?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
